### ISSUE TITLE
Now I know.

### ISSUE THREAD

**Issue by na3ee** [pi-node/instructions#146]
Windows updates must be paused.
Because rebooting will cause node problems...

**Comment by remzicirit** [comment]
yes it does. When windows restarts because of an updates seem whole thin is reset. I end up like you disable windows update but we need better solution without windows update  windows  will be at risk
here is an example because windows does not shutdown gracefully database  can get corrupted

2021-06-09 01:19:46 UTC [50-1] LOG:  database system was interrupted; last known up at 2021-06-08 23:37:23 UTC
2021-06-09 01:19:46 UTC [58-1] stellar@horizon FATAL:  the database system is starting up

It has to go recovery process. Here is an example
2021-06-09 01:20:15 UTC [50-2] LOG:  database system was not properly shut down; automatic recovery in progress
2021-06-09 01:20:15 UTC [50-3] LOG:  redo starts at 0/22771A8
2021-06-09 01:20:15 UTC [50-4] LOG:  invalid record length at 0/22782B0
2021-06-09 01:20:15 UTC [50-5] LOG:  redo done at 0/2278288
2021-06-09 01:20:15 UTC [50-6] LOG:  last completed transaction was at log time 2021-06-08 23:40:47.006187+00
2021-06-09 01:20:15 UTC [50-7] LOG:  MultiXact member wraparound protections are now enabled
2021-06-09 01:20:15 UTC [593-1] LOG:  autovacuum launcher started
2021-06-09 01:20:15 UTC [21-1] LOG:  database system is ready to accept connections
